internal/grpcutil/target.go

```diff
@@ -60,9 +60,17 @@ func ParseTarget(target string, skipUnixColonParsing bool) (ret resolver.Target)
 		return resolver.Target{Endpoint: target}
 	}
 	if ret.Scheme == "unix" {
+		// Prevents behavior change in "unix:///[...]" case.
+		if skipUnixColonParsing && ret.Authority == "" {
```

Comment:
Why does this check `skipUnixColonParsing`?  That only impacts `unix:<path>`, right?

Reply:
The check is still there just moved (and'd with the =="unix" check). This is so I don't append a "/" to the address when it isn't going to the resolver, since that is only so the resolver gets the right address.